debug_mode: true
max_playback_rate: 120

imu:
    device_name: GNSS_INS_SIM_IMU
    topic_name: /sim/sensor/imu
    allan_variance_curve:
        output_filename: /workspace/data/gnss_ins_sim/imu_calibration_results
        min_collection_time_in_mins: 50
        max_num_clusters: 100